### DISCUSSION TITLE
Error 404 ingreso curso

### DISCUSSION THREAD

**Discussion by JuanDavRH** [chamilo/chamilo-lms#5158] [General]
Buen dia,

Estoy presentando un problema, hace poco realice la actualización de chamilo de 1.11.12 a la 1.11.26, todo esta bien configurado aparentemente, los redireccionamientos están bien, pero cuando voy a ingresar a algun curso me aparece el siguiente error
Not Found
The requested URL was not found on this server.

Apache/2.4.41 (Ubuntu) Server at 20.34.46.13 Port 8082

**Comment by ywarnier** [comment]
Hola @JuanDavRH 
Es poca información para ayudarte. Normalmente Chamilo busca el "home" del curso como una redirección hacia main/course_home/ como puedes ver aquí https://github.com/chamilo/chamilo-lms/blob/53275c152275958b33a1f87a21843daa52fb543a/.htaccess#L26

Si te da un error 404 es (creo) porque tu Apache no está tomando en cuenta esta redirección o la está entendiendo mal. No parece ser un problema vinculado con 1.11.26 (por lo menos hasta obtener más datos). La migración de 1.11.12 a 1.11.26 no genera muchos cambios de estructura (solo se agregan unos archivos y se agregan algunas redirecciones), así que es poco probable que venga de Chamilo mismo. 

Quizás se haya actualizado Apache al mismo tiempo ?

**Comment by AyubxonNematov** [comment]
how fix this error 404 not found courses
